src/DonationForms/Properties/FormSettings.php

```diff
@@ -161,8 +161,13 @@ class FormSettings implements Arrayable, Jsonable
      */
     public $donateButtonCaption;
 
+    /**
+     * @var array
+     */
+    public $pdfSettings;
 
     /**
+     * @unreleased Add pdfSettings property
```

Comment:
you can remove this comment since its not released yet

Reply:
Resolved in 3116798be478233d2238a601cf98110b0d607b43